### PR TITLE
Current hotkey settings are now respected during the round

### LINKED ISSUE #250: [Feature] Remove chat toggle when leaving body as a ghost.
Leaving/entering body as a ghost toggles chat, didnt before, tg made it annoying.

### PR DESCRIPTION
**Pull request by X-TheDark**
fixes #250

Whenever you logged it, your hotkeys would be set according to what you set in the preferences, even if you change mobs/get cloned/etc during the round. This kind of sucks, so I changed it.

**Functionality changes**:
- When client connect to game (or reconnects), his hotkey settings are set to the ones in his preferences;
- If client changes his hotkey mode during the round, the game will respect that setting if the client changes mobs/ghosts/reenters corpse, etc; Your hotkeys will still be set to your preferences after the round ends (since you basically reconnect for the purposes of the game);

**Refactors**
- Moves client verbs to /modules/client/verbs;
- New hidden verb (because calling verbs with parameters is ass) for setting hotkeys (wonder if this should be a proc instead);

So yeah, best of both worlds. You get to say what your hotkey mode starts as and still can keep your current in-game hotkey settings during the round.

### CODE REVIEW

**Comment by Blukey** [comment]
i love you

**Comment by FluffySurvivor** [comment]
you can customise your hotkeys?

**Comment by X-TheDark** [comment]
No, it just won't switch your hotkey mode when you ghost/spawn-in-an-antag-spawner, etc. Currently, it uses your preferences (Game Preferences) hotkey options whenever you ghost/do something mob related. If you changed your mode in-game, it will reset it once you ghost and such.

File: code/modules/client/verbs/ooc.dm [hunk 1]
```diff
@@ -1,176 +1,176 @@
...
-	set desc ="Ignore a player's messages on the OOC channel"
-
-	var/selection = input("Please, select a player!", "Ignore", null, null) as null|anything in sortKey(clients)
-	if(!selection)
-		return
-	if(selection == src)
-		src << "You can't ignore yourself."
-		return
-	ignore_key(selection)
+/client/verb/ooc(msg as text)
```
Comment (oisin100) [inline code/modules/client/verbs/ooc.dm:1]: Github is weird...

**Comment by X-TheDark** [comment]
Ah, I see. Conflict because I moved my file to another directory, but Super added OOC tags (which I don't have, since it was merged after this PR was created) to OOC (which wasn't moved on his end), this conflict actually makes sense, for once.

**Comment by ShadowDeath6** [comment]
thank you

**Comment by X-TheDark** [comment]
Conflicts fixed.

**Comment by X-TheDark** [comment]
And fixed again.

**Pull request merged**